src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -91,6 +91,8 @@ MulticopterAttitudeControl::parameters_updated()
 						radians(_param_mc_yawrate_max.get())));
 
 	_man_tilt_max = math::radians(_param_mpc_man_tilt_max.get());
+
+	_attitude_input_filter.setParameters(1.f / (_param_mc_man_tilt_tau.get() * 0.707f), 0.707f);
```

Comment:
Half the square root of 2?

Kind of pedantic, but I think we should include the full thing rather than the magic number to help people answer their own questions.

Reply:
Good point, I'll at least make it a const variable

Reply:
PX4 defines got you covered:
https://github.com/PX4/PX4-Autopilot/blob/9be8f81d754d71a1357e739a21cac2c3d9be1acb/platforms/common/include/px4_platform_common/defines.h#L136